versions.csv

```diff
@@ -1,10 +1,10 @@
 AssemblyPath, Version
-iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.30.0
-iothub\service\src\Microsoft.Azure.Devices.csproj, 1.22.0
-shared\src\Microsoft.Azure.Devices.Shared.csproj, 1.20.1
-provisioning\device\src\Microsoft.Azure.Devices.Provisioning.Client.csproj, 1.6.0
-provisioning\transport\amqp\src\Microsoft.Azure.Devices.Provisioning.Transport.Amqp.csproj, 1.3.1
-provisioning\transport\http\src\Microsoft.Azure.Devices.Provisioning.Transport.Http.csproj, 1.2.4
-provisioning\transport\mqtt\src\Microsoft.Azure.Devices.Provisioning.Transport.Mqtt.csproj, 1.3.0
-security\tpm\src\Microsoft.Azure.Devices.Provisioning.Security.Tpm.csproj, 1.2.3
-provisioning\service\src\Microsoft.Azure.Devices.Provisioning.Service.csproj, 1.7.0
+iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.31.0-pnp-preview
+iothub\service\src\Microsoft.Azure.Devices.csproj, 1.23.0-pnp-preview
+shared\src\Microsoft.Azure.Devices.Shared.csproj, 1.21.0-pnp-preview
+provisioning\device\src\Microsoft.Azure.Devices.Provisioning.Client.csproj, 1.7.0-pnp-preview
+provisioning\transport\amqp\src\Microsoft.Azure.Devices.Provisioning.Transport.Amqp.csproj, 1.4.0-pnp-preview
```

Comment:
This is the first release post LTS -> minor bump (we reserve the LTS version for any patch fixes we might want to bring in).